.github/workflows/ci.yaml

```diff
@@ -4,6 +4,7 @@ on: [create, push]
 name: CI
 jobs:
   build:
+    if: ${{ github.event_name == 'push' || startsWith(github.ref, 'refs/tags/') }}
```

Comment:
I think this won't work on pull request events from personal forks of this repository (this PR is from the `unioslo` org) .  But I believe there are other problems preventing that as well (`github.repository_owner` should probably be `github.actor`, but only on PRs), so it can be addressed separately.